openedx.yaml

```
@@ -0,0 +1,24 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification
+
+---
+nick: codejail-includes
+tags:
+    - tools
```

Based on [see OEP-2](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0002-bp-repo-metadata.html#tags), this would be considered a "library" since it gets run in production.
```suggestion
    - library
```